recipes/pthread-stubs/meta.yaml

```diff
@@ -0,0 +1,29 @@
+{% set version = "0.1" %}
+{% set md5 = "774eabaf33440d534efe108ef9130a7d" %}
+
+package:
+  name: pthread-stubs
+  version: {{ version }}
+
+source:
+  fn: libpthread-stubs-{{ version }}.tar.bz2
+  url: http://xcb.freedesktop.org/dist/libpthread-stubs-{{ version }}.tar.bz2
+  md5: {{ md5 }}
+
+build:
+  skip: true  # [win or osx]
```

Comment:
I don't think this is needed for OS X because `libxbc` provides C bindings to access the X server API, and that's almost not used on Mac.